.travis.yml

```diff
@@ -51,35 +51,40 @@ matrix:
       env:
         - PLAT=aarch64
         - MB_ML_VER=2014
-    # causes a segfault
-    # - os: linux
-    #  arch: arm64
-    #  env:
-    #    - PLAT=aarch64
-    #    - INTERFACE64=1
-    #    - MB_ML_VER=2014
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
+     - os: linux
```

Comment:
There's an extra space before the `- os` that is causing the Travis CI yml parser to fail, so Travis CI never runs for this PR. Checked by pasting into their linter here: https://config.travis-ci.com/explore

Reply:
fixing